packages/backend/src/helpers/morgan.ts

```diff
@@ -26,15 +26,28 @@ morgan.token('cf-connecting-ip', (req: Request) => {
   }
 })
 morgan.token('graphql-query', (req: Request) => {
-  if (req.body.query) {
+  if (typeof req.body.query === 'string') {
     return req.body.query
       .replace(/\s+/g, ' ')
       .replace(/\n/g, '')
       .replace(/"/g, "'")
   }
 })
+
+const SENSITIVE_MUTATIONS = ['createConnection', 'updateConnection']
```

Comment:
nit: can we put this constant somewhere more discoverable - e.g. inside `graphql` directory as a new file called `sensitive-mutations.ts`, or `sensitive-graphql-list.ts` or similar?